README.md

```diff
@@ -52,7 +51,7 @@ Windows 10 (64-bit) with Python 3.6, 3.7 and 3.8.
 # Install Open3D stable release with pip
 $ pip install open3d
 
-# Install Open3D stable release with Conda
+# Install Open3D stable release with conda
 $ conda install -c open3d-admin -c conda-forge open3d
```

Comment:
We are deprecating conda packages. Let's just remove this and instead say:

    # Install Open3D stable release with pip (including in conda virtual environments)
    $ pip install open3d

Reply:
Done.